src/Servers/IIS/IIS/src/Microsoft.AspNetCore.Server.IIS.csproj

```
@@ -5,15 +5,14 @@
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <PackageId>Microsoft.AspNetCore.Server.IIS</PackageId>
     <Description>Provides support for hosting ASP.NET Core in IIS using the AspNetCoreModule.</Description>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;iis</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NativeAssetsTargetFramework>$(DefaultNetCoreTargetFramework)</NativeAssetsTargetFramework>
     <IsPackable>false</IsPackable>
 
     <!-- Ignore platform compatibility warnings for this project. We know this only works on windows.-->
-    <NoWarn>$(NoWarn);CA1416</NoWarn>
+    <NoWarn>CA1416</NoWarn>
```

I've been using `<NoWarn />` to clear the doc warning. It would help to bulk remove it if we can modify arcade.
```suggestion
    <NoWarn/>
    <NoWarn>$(NoWarn);CA1416</NoWarn>
```